scripts/qa_crowbarsetup.sh

```diff
@@ -901,11 +901,11 @@ function onadmin_set_source_variables
             CLOUDISONAME="SUSE-OPENSTACK-CLOUD-CROWBAR-9-${arch}-Media1.iso"
             CLOUDLOCALREPOS="SUSE-OpenStack-Cloud-Crowbar-9-devel"
         ;;
-        develcloud9)
```

Comment:
So we don't want develcloud9 anymore? Will that be put back later?

Reply:
It's still being here https://github.com/SUSE-Cloud/automation/pull/2708/commits/4a01c89cf5012ca587b2e12f2727219cbe8c33c8#diff-4da4ef75e4826a188f831da36089295cR1148

Reply:
@cmurphy due to a git rebase/merge artefact we had the develcloud9) case twice, its still there a few lines earlier.